src/entrypoint.sh

```diff
@@ -9,7 +9,7 @@ invoke () {
     then
         /usr/local/bin/invoke $@
     else
-        /usr/local/bin/invoke $@ >> /usr/src/{{project_name}}/invoke.log 2>&1
+        /usr/local/bin/invoke $@ >> /usr/src/{{project_name}}/celery.log 2>&1
```

Comment:
@jkariukidev why did you rename this to `celery.log`? It's the log of the tasks invocation, which is executed both from the `django` and the `celery` containers. 
No need to change things not related to the issue...